pre-commit.template

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
+#
+# PreReq's
+#  - This requires that a current level of Eclipse is defined on your machine 
+
+# This hook will run the eclipse code formatter before any commit.
+# This will format Java code files that are part of the commit.
+# The java code will be formatted based on the preferences defined
+# in the java.perfs file provided in the base directory of this project.
+#
+# Update the following items below then save this file in 
+# .git/hooks/pre-commit 
+# be sure you have execute authority for this file. 
+
+# set this to the location of your Eclipse installtion
+# Mac OSX Example: /Users/mac/EclipseInstalls/EclipseO2/EClipse.app/Contents/MacOS
+ECLIPSE_HOME=**********Fully qualified location of Eclipse executable ******
+
+# Style file is provided in the root directory of the project as java.perfs
+# The project location is the location of the boost-maven project on your
+# workstation. 
+# example: /Users/mac/OpenLiberty-boost-maven/boost-maven
+PROJECT_LOCATION=********Fully qualified location of this projects .git folder *****
+STYLE_FILE=$PROJECT_LOCATION/java.perfs
```

Comment:
Should this be named `java.prefs`?

Reply:
Open to suggestions. how does javaFormat.perfs sound?

Reply:
Oh, I just meant that the extension should be `prefs` (preferences) instead of `perfs`. Probably just a typo :)

Reply:
Ah!! My dyslexia strikes again!. I'll update.